contrib/test_jenkins.sh

```diff
@@ -444,6 +444,35 @@ run_ucx_perftest_mpi() {
 }
 
 #
+# Run UCX CUDA performance test with MPI
+#
+run_ucx_perftest_cuda_mpi() {
```

Comment:
is it possible to just add "-m cuda" to contrib/test_types, so run_ucx_perftest_mpi() will run it, instead of adding new bash function?